Alignment/CTPPS/data/RPixGeometryCorrections.xml

```diff
@@ -0,0 +1,16 @@
+<xml DocumentType="AlignmentDescription">
```

Comment:
What is the usage of this file? Why this is not in db?

Reply:
This is the file storing the parameters to align the pixel planes wrt each other. An internal local alignment that does not change if you don't change the detector package itself. It is not in the DB because it changes only if you change the detector packages.